WarehouseManagement.Core/DTOs/Delivery/DeliveryDto.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace WarehouseManagement.Core.DTOs.Delivery;
+
+public class DeliveryDto
+{
+    public int Id { get; set; }
+    public string SystemNumber { get; set; } = string.Empty;
+    public string ReceptionNumber { get; set; } = string.Empty;
```

Comment:
these can be more than one.

Reply:
I don't see how can you set more than one on these properties since the entity class Delivery has single SystemNumber and single ReceptionNumber. If there is a way to do that I will need help with it.

public class Delivery : BaseClass
{
    public string SystemNumber { get; set; } = string.Empty;
    public string ReceptionNumber { get; set; } = string.Empty;
    public string TruckNumber { get; set; } = string.Empty;

Reply:
cant we just have it as a list/array of strings? thats what i meant.

Reply:
I am sorry Kris, but I don't get it. The entity Model has Single ReceptionNumber and Single SystemNumber. Why  would we have List<string> SystemNumbers and List<string> ReceptionNumbers?

Reply:
The client has said that a delivery can have more than one SystemNumber and ReceptionNumber. This one is just for the Reception Number but he has also told me that it can have more than 1 system number.
![image](https://github.com/InterCarsBulgaria-TechLeadAcademy/warehouse.management.be/assets/18427030/a20db439-32ff-4690-a433-169845466ed5)

Reply:
Ok, to do this we need to change the Db structure, tto be able to have multiple SystemNumbers and ReceptionNumbers in the delivery entity. We will need mapping tables for both properties in order to have more than one value. I suggest you to implement this logic in a separate branch. Lets not play with the DB in this branch.

Reply:
ok, didnt know that we had made the DB like that. Yes, lets do it in a separate branch.